core/src/test/scala/org/apache/spark/sql/catalyst/plans/BasePlanTest.scala

```diff
@@ -159,24 +159,4 @@ class BasePlanTest extends BaseTiSparkTest {
     assert(estimatedRowCount === answer)
   }
 
-  test("baseline test") {
```

Comment:
Why do we move this test?

Reply:
I think `test` should belong to a `suite` rather than `BasexxxTest`